### PR TITLE
Convert all demos on docs to lite

### PR DESCRIPTION
**Pull request by aliabd**
- Converts the demos on both main and pip docs to lite. 
- Removes the demos for 3.x docs (keeps the code snippets)
- Removes the embedded component for now, will bring it back with lite in a separate PR. 

I will still need to differentiate the main and pip demos, which currently both use the released version of lite. I'll upload the main wheel of lite to aws and add it to our github actions workflow. 

But this currently finally makes the super annoying docs issue go away where embedded spaces were interfering with the page and causing the js to break/users to see previous docs content on a new page. Please test this by navigating really fast between pages. 

Closes: #4922 #5825

### CODE REVIEW

**Comment by abidlabs** [comment]
awesome! closes https://github.com/gradio-app/gradio/issues/4922 and maybe closes https://github.com/gradio-app/gradio/issues/5825 as well?

**Comment by abidlabs** [comment]
Very cool @aliabd! Went through a bunch of demos and they all look good to me!

File: js/_website/src/lib/components/Demos.svelte [hunk 1]
```diff
@@ -26,33 +41,11 @@
 				Open in 🎢 ↗
 			</button>
 
-			<gradio-lite playground shared-worker class="p-2">
+			<gradio-lite playground shared-worker layout="vertical" class="p-2">
```
Comment (freddyaboulton) [inline js/_website/src/lib/components/Demos.svelte:44]: Annoying suggestion, but can the demo go before the code? Users now have to scroll down a lot to see the demo.

playground_lite_docs

**Review by freddyaboulton — APPROVED** [review]
Nice PR @aliabd ! Are we also replacing the demos in the guides with lite?

File: js/_website/src/lib/components/Demos.svelte (hunk 1 above)
Comment (aliabd) [reply js/_website/src/lib/components/Demos.svelte:44]: Good point that's kind of annoying. I'll fix it on the lite playground side and not directly on the website. Opened an issue https://github.com/gradio-app/gradio/issues/8228

**Comment by aliabd** [comment]
@freddyaboulton yes, will do that in a separate PR because it's slightly different. Most of it is written content anyway (ie will just replace `<gradio-app space="...">` with `<gradio-lite>{code}</gradio-lite>`)

**Pull request merged**